core/math/quat.h

```diff
@@ -71,6 +71,26 @@ class _NO_DISCARD_CLASS_ Quat {
 		r_axis.z = z * r;
 	}
 
+	void set_rotation(const Vector3 &p_from, const Vector3 &p_to) {
```

Comment:
Should these be unit vectors? What happens if non-unit or zero?

If unit input, should there be math checks (at least in dev build)?

Reply:
I'm not familiar with the math :P

Reply:
Ah, this is common with copy pastas. :smile: I guess math checks could be a separate PR.